StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerAdapter/CustomerAdapter.swift

```diff
@@ -143,11 +143,16 @@ open class StripeCustomerAdapter: CustomerAdapter {
                 using: customerEphemeralKey.ephemeralKeySecret,
                 types: savedPaymentMethodTypes
             ) { paymentMethods, error in
-                guard let paymentMethods = paymentMethods, error == nil else {
+                guard var paymentMethods, error == nil else {
                     let error = error ?? PaymentSheetError.unexpectedResponseFromStripeAPI // TODO: make a better default error
                     continuation.resume(throwing: error)
                     return
                 }
+                // Remove cards that originated from Apple or Google Pay
+                paymentMethods = paymentMethods.filter { paymentMethod in
```

Comment:
On Android we pass these into a higher level object/method which actually makes the API call. Should we do the filtering there?

Reply:
Talked about it offline - we don't have that higher level object yet, but feels reasonable to do that!  I won't do that in this PR though.

Reply:
Chatted offline, doesn't make sense for iOS to do right now since they don't have the same abstractions as Android.